<?xml version="1.0"?>
<launch>

  <group ns="cpp">
    <node name="timer_test_cpp" pkg="timer_test" type="timer_test_cpp"
      output="screen" >
    </node>

    <node name="average_dt" pkg="topic_state" type="average.py"
      output="screen" >
      <remap from="topic" to="update_dt" />
    </node>
  </group>

  <group ns="python">
    <node name="timer_test_py" pkg="timer_test" type="timer_test.py"
      output="screen" >
    </node>

    <node name="average_dt" pkg="topic_state" type="average.py"
      output="screen" >
      <remap from="topic" to="update_dt" />
    </node>
  </group>

</launch>
